__tests__/app.test.js

```diff
@@ -307,20 +307,10 @@ describe("POST: /api/articles/:article_id/comments", () => {
 })
 
 describe("DELETE /api/comments/:comment_id", () => {
-    test("200: Successfully deletes comment and returns deleted comment", () => {
+    test("204: Successfully deletes comment and returns no content", () => {
```

Comment:
Excellent and yes because its a 204 we are also unable to send content as a result as well